.gitignore

```diff
@@ -9,3 +9,4 @@ venv/
 *.pyc
 app/db.sqlite3
 db.sqlite3
+.venv
```

Comment:
There is an extra space after `.venv`. It should be removed to ensure `.venv` is correctly ignored by Git.